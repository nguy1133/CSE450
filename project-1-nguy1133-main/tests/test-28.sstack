// SPLIT should work when the second argument is on the stack.
// Also: // should be allowed inside of strings.
PUSH "http://www.cse.msu.edu/~cse450/"
SPLIT "//"
PRINT
PRINT
