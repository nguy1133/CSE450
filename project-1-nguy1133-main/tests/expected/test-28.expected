www.cse.msu.edu/~cse450/
http:
